### ISSUE TITLE
[BUG] cannot download user or saved posts; authentication requirement not communicated to the user

### ISSUE THREAD

**Issue by Zireael07** [Serene-Arc/bulk-downloader-for-reddit#880]
- [X] I am reporting a bug.
- [X] I am running the latest version of BDfR
- [X] I have read the [Opening an issue](https://github.com/aliparlakci/bulk-downloader-for-reddit/blob/master/docs/CONTRIBUTING.md#opening-an-issue)

## Description

bdfr download -u xxxx -v ./reddit doesn't download anything

adding -v reveals that it's using unaunthenticated instance (and it never asked for authentication unlike what README says)

## Command

See above

## Environment (please complete the following information)

- OS: [Linux Manjaro
- Python version: 3.10

## Logs

Nothing to paste because that initial run didn't give anything in log, and now I've run it again properly, there is no longer the initial log...

**Comment by Fakeaccount12312** [comment]
What you are doing is downloading the posts made by the specified user. To download the saved posts, use --saved. Of course you need to authenticate because your saved posts aren't visible to anyone except you.

**Comment by Zireael07** [comment]
Yes, I know. But neither trying to download user's posts nor saved posts does anything.

The thing is, it never prompts me to authenticate. Without -v there is literally ZERO feedback to end user what's going wrong.

By trial and error, I found out reversing the parameters bdfr clone ./reddit -u xxxx DOES prompt for authenticate parameter. When I gave it, and followed the instructions, it finally worked

README claims the order is bdfr [options] [output] and so does --help.

Also README claims "most users won't need authentication" which isn't true as wanting to download saved posts is how many people learn this tool exists.

**Comment by Fakeaccount12312** [comment]
Maybe change the title to be more indicative of the problem then.

**Comment by Zireael07** [comment]
Changed the title now that I know more about what happened
